tests/integration_tests/Test/PM/Integration/Health.cls

```diff
@@ -0,0 +1,43 @@
+Class Test.PM.Integration.Health Extends Test.PM.Integration.Base
+{
+
+Method TestHealthTrue()
```

Comment:
On further thought, rather than $$$AssertSkipped, should $$$AssertStatusNotOK on IRIS (not for Health)

Reply:
Does that mean we should expect failed test cases on incompatible platforms?

Reply:
If so, can we just remove the platform check? Since `zpm "load /path/to/incompatible/module"` will automatically fail in the following step

Reply:
On an incompatible platform, the test should pass iff zpm "load" fails.